src/WebJobs.Script/Diagnostics/OpenTelemetry/ResourceSemanticConventions.cs

```diff
@@ -14,14 +14,13 @@ internal static class ResourceSemanticConventions
         internal const string CloudProvider = "cloud.provider";
         internal const string CloudPlatform = "cloud.platform";
         internal const string CloudRegion = "cloud.region";
-        internal const string CloudResourceId = "cloud.resource.id";
+        internal const string CloudResourceId = "cloud.resource_id";
 
         //FaaS
         internal const string FaaSTrigger = "faas.trigger";
         internal const string FaaSInvocationId = "faas.invocation_id";
         internal const string FaaSColdStart = "faas.coldstart";
         internal const string FaaSName = "faas.name";
-        internal const string FaaSVersion = "faas.version";
```

Comment:
Did we need to remove this? Can we use this to track the host version?

Reply:
I am not certain, but I _think_ it is redundant with `service.version` attribute

Reply:
yes, I will remove it in the next PR. It's no longer appliable. https://opentelemetry.io/docs/specs/semconv/resource/faas/#:~:text=Azure%20Functions%3A%20Not%20applicable.%20Do%20not%20set%20this%20attribute.